project-templates/import-records/Makefile

```diff
@@ -0,0 +1,102 @@
+
+SHELL=/bin/sh -o pipefail
+export PATH := ./node_modules/.bin/:$(PATH)
+
+DIST_DIR ?= dist
+EXTRA_IMPORT_FLAGS ?= -d
+EXTRA_IMPORT_FILTERS ?= head -2
+PLACES_LEVEL_0 ?= places-level-0.csv
+PLACES_LEVEL_0_DIST ?= places-level-0.csv
+PLACES_LEVEL_1 ?= places-level-1.csv
+PLACES_LEVEL_1_DIST ?= places-level-1.csv
+PLACES_LEVEL_2 ?= places-level-2.csv
+PLACES_LEVEL_2_DIST ?= places-level-2.csv
+PLACES_LEVEL_3 ?= places-level-3.csv
+PLACES_LEVEL_3_DIST ?= places-level-3.csv
+PLACES_LEVEL_3_FIELDMAP ?= places-level-3-fieldmap.csv
+PLACES_LEVEL_3_PEOPLE_FIELDMAP ?= places-level-3-people-fieldmap.csv
+PEOPLE ?= people.csv
+PEOPLE_DIST ?= people.csv
+RECORDS ?= records.csv
+RECORDS_DIST ?= records.csv
+RECORDS_FIELDMAP ?= records-fieldmap.csv
+
+#all: init $(PLACES_LEVEL_1_DIST) $(PLACES_LEVEL_2_DIST) $(PLACES_LEVEL_3_DIST) $(PEOPLE_DIST)
+all: init $(DIST_DIR)/$(RECORDS_DIST)
+
+.PHONY: init
+
+init:
+	test -d $(DIST_DIR) || mkdir $(DIST_DIR)
+	test -d tmp || mkdir tmp
+
+#$(PLACES_LEVEL_1_DIST): $(PLACES_LEVEL_1)
+#	# Just copying because these places were created manually.
+#	cp $(PLACES_LEVEL_1) $(PLACES_LEVEL_1_DIST)
+#
+#$(PLACES_LEVEL_2_DIST): $(PLACES_LEVEL_2)
+#	cat $(PLACES_LEVEL_2) | \
+#	  medic-format-csv -c places-level-2-config.js \
+#	    > $(PLACES_LEVEL_2_DIST)
```

Comment:
Yeah in this specific example project I didn't need to create these levels of places because they were created manually.  But I left them commented in because there are some cases where you need to create them too.